.bowerrc

```diff
@@ -1,3 +1,4 @@
 {
-  "directory": "app/assets/bower_components"
+  "directory": "app/assets/bower_components",
+  "interactive": false
```

Comment:
@jbrooksuk We need to find a way to get bower to auto-select the correct version of jquery here too.